qcodes/instrument_drivers/Kepco/KepcoMagnet.py

```diff
@@ -0,0 +1,65 @@
+from qcodes.instrument.base import Instrument
+from qcodes.utils.validators import Numbers
+import qcodes as qc
+
+from typing import Union
+
+class KepcoMagnet(Instrument):
+    MAX_AMP = 20
+
+    """
+    This is the qcodes driver for controlling the field using the Kepco BOP 20-20M.
+
+    This is a virtual driver only and will not talk to your instrument.
```

Comment:
It would be good to explain a bit more that this actually controls a different parameter. By this description I would expect that this is a control for a instrument which can only be controlled manually